utility/src/test/java/org/oppia/android/util/parser/html/PolicyPageTagHandlerTest.kt

```diff
@@ -0,0 +1,187 @@
+package org.oppia.android.util.parser.html
+
+import android.app.Application
+import android.content.Context
+import android.text.Html
+import android.text.Spannable
+import android.text.style.ClickableSpan
+import android.widget.TextView
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import dagger.Binds
+import dagger.BindsInstance
+import dagger.Component
+import dagger.Module
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentCaptor
+import org.mockito.Captor
+import org.mockito.Mock
+import org.mockito.Mockito.times
+import org.mockito.Mockito.verify
+import org.mockito.junit.MockitoJUnit
+import org.mockito.junit.MockitoRule
+import org.oppia.android.app.model.PolicyPage
+import org.oppia.android.testing.mockito.capture
+import org.oppia.android.testing.robolectric.RobolectricModule
+import org.oppia.android.testing.threading.TestDispatcherModule
+import org.oppia.android.testing.time.FakeOppiaClockModule
+import org.oppia.android.util.locale.LocaleProdModule
+import org.oppia.android.util.logging.ConsoleLogger
+import org.oppia.android.util.logging.LoggerModule
+import org.oppia.android.util.parser.html.CustomHtmlContentHandler.CustomTagHandler
+import org.oppia.android.util.parser.html.PolicyPageTagHandler.PolicyPageLinkClickListener
+import org.robolectric.annotation.LooperMode
+import javax.inject.Inject
+import javax.inject.Singleton
+import kotlin.reflect.KClass
+
+private const val POLICY_PAGE_LINK_MARKUP_1 =
+  "By using %s, you agree to our <br> <oppia-noninteractive-policy link=\"tos\"> Terms of " +
+    "Service </oppia-noninteractive-policy> and <oppia-noninteractive-policy link=\"privacy\">" +
+    "Privacy Policy </oppia-noninteractive-policy>."
+
+/** Tests for [PolicyPageTagHandler]. */
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+class PolicyPageTagHandlerTest {
+  @Rule
+  @JvmField
+  val mockitoRule: MockitoRule = MockitoJUnit.rule()
+
+  @Mock
+  lateinit var mockImageRetriever: FakeImageRetriever
+
+  @Mock
+  lateinit var mockPolicyPageLinkClickListener: PolicyPageLinkClickListener
+
+  @Captor
+  lateinit var policyPageCaptor: ArgumentCaptor<PolicyPage>
+
+  @Inject
+  lateinit var context: Context
```

Comment:
Nit: can fit on the line above for consistency with ``consoleLogger``.

Reply:
Done.